### ISSUE TITLE
No refi boot after you restart your Mac 

### ISSUE THREAD

**Issue by wsyw1234** [BenSova/Patched-Sur#317]
**Describe the bug**
When I restart my Mac I only have install Big Sur option but don't have EFI Boot option. The yellow usb logo never shows up at all.

**Desktop (please complete the following information):**
 - Upgrading from: [Catalina v10.15.7]
- Upgrading to: [Big Sur v11.]
 - Mac Model: [13" 2012 MacBook Pro]
 - Version [v0.0.7]

**Comment by ozziepadilla** [comment]
Did hold down the OPTIOM key while turn it in your Mac 💻?

**Comment by wsyw1234** [comment]
> Did you hold down the OPTION key while turn it on your Mac 💻?

Yes I did, it only show my original system and install big sur but didn't show efi boot

**Comment by ozziepadilla** [comment]
Are you using a usb hub or you plug in tue usb directly to your Mac 💻?

**Comment by wsyw1234** [comment]
> Are you using a usb hub or you plug in the usb directly to your Mac 💻?

I plug in the USB directly to my mac

**Comment by wsyw1234** [comment]
![20210220_191450](https://user-images.githubusercontent.com/79382144/108611779-f0b81900-73af-11eb-9265-a923116a3b49.jpg)
![20210220_191354](https://user-images.githubusercontent.com/79382144/108611780-f150af80-73af-11eb-91d1-3031cce0eb82.jpg)

**Comment by wsyw1234** [comment]
> Are you using a usb hub or you plug in the usb directly to your Mac 💻?

I just updated pictures

**Comment by ozziepadilla** [comment]
Wait for benSova to reply your issues I was looking for a solution but didn’t find why you are having this inconvenience

**Comment by wsyw1234** [comment]
> Wait for benSova to reply your issues I was looking for a solution but didn’t find why you are having this inconvenience

thank you！

**Comment by wsyw1234** [comment]
> Wait for benSova to reply your issues I was looking for a solution but didn’t find why you are having this inconvenience

if you have any news  please contact me 🙏 thank you

**Comment by masriyan23** [comment]
I also have a problem there

> **Describe the bug**
> When I restart my Mac I only have install Big Sur option but don't have EFI Boot option. The yellow usb logo never shows up at all.
> 
> **Desktop (please complete the following information):**
> 
> * Upgrading from: [Catalina v10.15.7]
> * Upgrading to: [Big Sur v11.]
> * Mac Model: [13" 2012 MacBook Pro]
> * Version [v0.0.7]

I also have a problem there
